gosuper/gosuper/api.go

```diff
@@ -236,6 +237,15 @@ func LogToDisplayControl(writer http.ResponseWriter, request *http.Request) {
 		return
 	}
 
+	if loaded, err := systemd.Dbus.GetUnitProperty(serviceName, "LoadState"); err != nil {
+		sendError(fmt.Errorf("Unable to get log to display load status: %v", err))
+		return
+	} else if loaded.Value.String() == `"not-found"` {
```

Comment:
True, but so far this action only makes sense on Resin OS, where we can keep track of things changing like that  (systemd versions changing, or use of locales)